### PR TITLE
FSS-11 # Add region to response

### PR DESCRIPTION
**Pull request by kizaonline**

### CODE REVIEW

File: lib/credentials.js [hunk 1]
```diff
@@ -3,11 +3,13 @@
 
 const AWS = require('aws-sdk')
 const uuidV4 = require('uuid/v4')
+const path = require('path')
+// $FlowFixMe
+const config = require(path.join(__dirname, '..', '.blinkmrc.json'))
```
Comment (mymattcarroll) [inline lib/credentials.js]: Did this not work just using:

```javascript
const config = require('../.blinkmrc.json')
```

Comment (mymattcarroll) [inline lib/credentials.js]: Just curious, did this not work using:

```javascript
const config = require('../.blinkmrc.json')
```

**Comment by mymattcarroll** [comment]
LGTM 👍

File: lib/credentials.js (hunk 1 above)
Comment (kizaonline) [inline lib/credentials.js]: Yes thats correct, I couldnt find anyway to delimit the leading dot and instead had to use path

**Pull request merged**